lib/org/chromium/webidl/ast/MapLike.js

```diff
@@ -21,6 +21,11 @@ foam.CLASS({
   ],
 
   methods: [
+    function getName() {
+      var keyType = this.type ? this.type.getName() : null;
+      var valueType = this.valueType ? this.valueType.getName() : null;
+      return `maplike<${keyType}, ${valueType}>`;
```

Comment:
For maplike, the name will look like its definition as well (e.g. `maplike<String, String>`)